mandatory/1-syntax-errors.js

```diff
@@ -25,16 +25,16 @@ To run the tests for just this one file, type `npm test -- --testPathPattern 1-s
 ===================================================
 */
 
-test("addNumbers adds numbers correctly", () => {
-  expect(addNumbers(3, 4, 6)).toEqual(13);
-});
+test('addNumbers adds numbers correctly', () => {
+  expect(addNumbers(3, 4, 6)).toEqual(13)
+})
 
-test("introduceMe function returns the correct string", () => {
-  expect(introduceMe("Sonjide", 27)).toEqual(
-    "Hello, my name is Sonjide and I am 27 years old"
-  );
-});
+test('introduceMe function returns the correct string', () => {
+  expect(introduceMe('Sonjide', 27)).toEqual(
+    'Hello, my name is Sonjide and I am 27 years old',
+  )
+})
 
-test("getTotal returns a string describing the total", () => {
-  expect(getTotal(23, 5)).toEqual("The total is 28");
-});
+test('getTotal returns a string describing the total', () => {
+  expect(getTotal(23, 5)).toEqual('The total is 28')
+})
```

Comment:
I assume you didn't mean to change all these tests, I assume from a change all? Be careful when doing this when tests are in the same file (though in the real world that wouldn't be case!)